lib/tracker_api/resources/search_result_container.rb

```diff
@@ -0,0 +1,12 @@
+module TrackerApi
+  module Resources
+    class SearchResultContainer
```

Comment:
Probably makes sense to move the `attribute :id` and `Equalizer` code into its own subclass of Base so classes that don't need an `id` don't have to have one. I think it is fine to leave for now since it doesn't hurt anything and I am working on a major refactor that will change much of this anyway. So not worth the effort now.